rustls/src/client/handy.rs

```diff
@@ -69,119 +71,239 @@ mod cache {
         }
     }
 
-    /// An implementer of `ClientSessionStore` that stores everything
```

Comment:
yuck. This diff needs splitting so the lift into a module is separate from the creation of the no-std version. I'll adjust.

Reply:
Split up.